modules/ts/src/ts_perf.cpp

```diff
@@ -1691,6 +1943,10 @@ void TestBase::RunPerfTestBody()
 {
     try
     {
+#ifdef ENABLE_INSTRUMENTATION
+        cv::instr::setFlags(cv::instr::getFlags()|cv::instr::FLAGS_MAPPING); // enable mapping for first run
+        cv::instr::queueCommand(cv::instr::COMMAND_RESET); // delayed clean up. actual execution before timer start
```

Comment:
I believe, this code should help to call "reset" once:
```
bool TestBase::startTimer()
{
#ifdef ENABLE_INSTRUMENTATION
    if (currentIter == 0)
        cv::instr::resetTrace();
#endif
...
}
```

Queue/process pattern look to me over-complicated here.